avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -73,6 +73,22 @@ unit =
 ##relTh = 0.75|0.9|0.8
 
 
+[PLOTS]
+# if extraPlots true additional analysis plots are created
+extraPlots = True
+# for one to one comparison of two result variables or derived quantities with option to distinguish scenarios using scenario
+# comparison result variables options: resTypeFieldMax (or Min or Mean), maxresTypeCrossMax, sRunout, deltaSXY, zRelease,
+# zRunout, deltaH, relMass, finalMass, entMass, thalwegTravelAngle
+# comparison result variable 1
+compResType1 = pfvFieldMax|deltaSXY
+# comparison result variable 2
+compResType2 = pftFieldMax|runoutAngle
+# scenario parameter name
```

Comment:
What is this parameter for? I think a better description is needed

Reply:
added more info